romtools/vector_space/utils/scaler.py

```diff
@@ -169,12 +171,29 @@ def post_scale(self, data_tensor: np.ndarray) -> np.ndarray:
 
         Returns:
             np.ndarray: The scaled data matrix.
-        '''
+        """
         return self.__scaling_vector_matrix[None, :, None] * data_tensor
 
 
+class ScalarScaler:
+    """
+    Applies a scalar scale factor
+
+    This class conforms to `Scaler` protocol.
+    """
+
+    def __init__(self, factor: float = 1.0) -> None:
+        self._factor = factor
+
+    def pre_scale(self, data_tensor: np.ndarray) -> np.ndarray:
+        return data_tensor / self._factor
+
+    def post_scale(self, data_tensor: np.ndarray) -> np.ndarray:
+        return data_tensor * self._factor
```

Comment:
i thought we were doing operations in place, is that not true?

Reply:
I think we did decide that, but all of the other scalers look like they return a copy so maybe that didn't make its way down to the scalers?

Reply:
Yes, it looks like there are some inconsistencies, I'll make an issue.